gudrun_classes/sample.py

```diff
@@ -11,8 +9,83 @@
 
 
 class Sample:
+    """
+    Class to represent a Sample.
+
+    ...
+
+    Attributes
+    ----------
+    name : str
+        Name of the sample.
+    numberOfFilesPeriodNumber : tuple(int, int)
+        Number of data files and their period number.
+    dataFiles : DataFiles
+        DataFiles object storing data files belonging to the container.
+    forceCalculationOfCorrections : bool
+        Decide whether to force calculations of corrections or read them
+        from the file, if it exists.
+    composition : Composition
+        Composition object storing the atomic composition of the sample.
+    geometry : str
+        Geometry of the sample (FLATPLATE / CYLINDRICAL).
+    thickness : tuple(float, float)
+        Upstream and downstream thickness.
+    angleOfRotationSampleWidth : tuple(float, float)
+        Angle of rotation of the sample and its width.
+    densityOfAtoms : str
+        Density of atoms in the sample (atoms/Angstrom^3)
+    overallBackgroundFactor : float
+        Background factor.
+    totalCrossSectionSource : str
+        TABLES / TRANSMISSION monitor / filename
+    sampleTweakFactor : float
+        Tweak factor for the sample.
+    topHatW : float
+        Width of top hat function for Fourier transform.
+    minRadFT : float
+        Minimum radius for Fourier transform.
+    gor : float
+        Broadening power on g(r).
+        0 = constant, 0.5 = sqrt(r), 1 = r
```

Comment:
```suggestion
    gor : float
        Broadening of g(r) at r = 1 Angstrom
```
You might consider giving this a better name - `grBroadening` perhaps?

Reply:
Yes makes sense. I didn't realise this attributes functionality until my trawl through the source code..